lib/chat_message.dart

```diff
@@ -17,6 +19,15 @@ class ChatMessage {
     );
   }
 
+  static List<ChatMessage> fromQueryDocumentSnapshot(
```

Comment:
nit: I'm not sure if it's a good idea to put this logic in this class.

Reply:
what do you mean? where should i put?

Reply:
Handling the List of `ChatMessage` in the ChatMessage class looks weird to me. I meant probably you can just pull out of the class.

Reply:
Again. It's a nit. not important.

Reply:
yes you're right, but I don't have place to put

Reply:
I was thinking of just putting it in the `main.dart` but not a big deal. Just leave it here. Maybe adding a comment might be a good idea if you decided to leave it here.